test/spec/sanity-checks.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env mocha */
+
+var request = require('supertest')
+var assert = require('assert')
+var path = require('path')
+var fs = require('fs')
+
+/**
+ * Basic sanity checks on the dev server
+ */
+describe('The prototype kit', function () {
+  var app
+  var server
+
+  before(function () {
+    app = require('../../server')
+    server = app.listen(3000)
```

Comment:
I don't think the server needs to be listening at this point. `supertest` can just accept the `app` variable and handles the listening and un-listening itself. So you can delete `var server`, `server = app.listen(3000)` and the entire `after()` hook and it should still run